packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -205,6 +207,29 @@ export class Organization extends BaseResource implements OrganizationResource {
       .catch(() => []);
   };
 
+  getInvitations = async (
+    getInvitationsParams?: GetInvitationsParams,
+  ): Promise<ClerkPaginatedResponse<OrganizationInvitationResource>> => {
+    return await BaseResource._fetch({
+      path: `/organizations/${this.id}/invitations`,
+      method: 'GET',
+      search: convertPageToOffset(getInvitationsParams) as any,
+    })
+      .then(res => {
+        const { data: requests, total_count } =
+          res?.response as unknown as ClerkPaginatedResponse<OrganizationInvitationJSON>;
+
+        return {
+          total_count,
+          data: requests.map(request => new OrganizationInvitation(request)),
+        };
+      })
+      .catch(() => ({
```

Comment:
🔧 Catching the error at this point prevents the global error handler of the FAPI client to kick-in. Is there a specific reason we want to do this?

Reply:
It seems we used it previously for any method that was fetching arrays

- https://github.com/clerkinc/javascript/blob/df4f686b64e790039fb301278efb637cb1c7efba/packages/clerk-js/src/core/resources/Organization.ts#L193
- https://github.com/clerkinc/javascript/blob/df4f686b64e790039fb301278efb637cb1c7efba/packages/clerk-js/src/core/resources/SessionWithActivities.ts#L54

- [getMemberships before paginated responces ](https://github.com/clerkinc/javascript/blob/972f24faed097cd642f900c4dd6cb492810a1f47/packages/clerk-js/src/core/resources/Organization.ts#L73)

Reply:
I see. I am not sure if it's correct thought. We should the error bubble up.

Reply:
Wouldn't be breaking change to change it now ? Shall we aim to fix for v5 ?